website/docs/sdk-reference/elixir.md

```diff
@@ -0,0 +1,250 @@
+value = ConfigCat.get_value("key", "my default value") # Returns "my default value"
+ConfigCat.force_refresh();
+value = ConfigCat.get_value("key", "my default value") # Returns "value from server"
+```
+
+### Custom cache behaviour with `cache:` option parameter
+
+To be able to customize the caching layer you need to implement the following behaviour:
+
+```elixir
+defmodule ConfigCat.ConfigCache do
+  alias ConfigCat.Config
+
+  @type key :: String.t()
+  @type result :: {:ok, Config.t()} | {:error, :not_found}
+
+  @callback get(key) :: {:ok, Config.t()} | {:error, :not_found}
+  @callback set(key, Config.t()) :: :ok
+end
+```
+
+- You must implement (either explicitly or implicitly) the ConfigCache behaviour
+- It is the responsibility of the calling application to supervise the cache if it needs to be supervised.
+
+### Multiple `ConfigCat` instances
+
+If you need to run more than one instance of `ConfigCat`, you can add multiple
+`ConfigCat` children. You will need to give `ConfigCat` a unique `name` option
+for each, as well as using `Supervisor.child_spec/2` to provide a unique `id`
+for each instance.
+
+```elixir
+# lib/my_app/application.ex
+def start(_type, _args) do
+  children = [
+    # ... other children ...
+    Supervisor.child_spec({ConfigCat, [sdk_key: "sdk_key_1", name: :first]}),
+    Supervisor.child_spec({ConfigCat, [sdk_key: "sdk_key_2", name: :second]}),
+  ]
+  opts = [strategy: :one_for_one, name: MyApp.Supervisor]
+  Supervisor.start_link(children, opts)
+end
+```
+
+Then you can call `.get_value/4` like this:
+
+```elixir
+ConfigCat.get_value("setting", "default", client: :first)
+ConfigCat.get_value("setting", "default", client: :second)
+```
+
+## Logging
+
+In the *ConfigCat SDK*, we use the default Elixir's [Logger](https://hexdocs.pm/logger/Logger.html) so you can customise as you like.
+
+```bash
+[info]  Fetching configuration from ConfigCat
+[info]  ConfigCat configuration json fetch response code: 200 Cached: "xxxxx-xxx"
+[debug] Evaluating get_value('show_contact_phone_number').
+```
+
+## `get_all_keys()`
+
+You can query the keys from your configuration in the SDK with the `get_all_keys()` method.
+
+```elixir
+keys = ConfigCat.get_all_keys()
+```
+
+## Using ConfigCat behind a proxy
+
+Provide your own network credentials (username/password), and proxy server settings (proxy server/port) by passing the proxy details to the creator method.
+
+```elixir
+{ConfigCat, [
+    sdk_key: "YOUR SDK KEY",
+    http_proxy: "https://user@pass:yourproxy.com"
+]}
+```
+
+## Sample Applications
+
+- <a href="https://github.com/configcat/elixir-sdk/tree/master/samples/" target="_blank">Sample App</a>
+
+## Look under the hood
+
+- <a href="https://hexdocs.pm/configcat" target="_blank">ConfigCat's HexDocs</a>
+- <a href="https://github.com/configcat/elixir-sdk" target="_blank">ConfigCat's Elixir SDK on GitHub</a>
+- <a href="https://hex.pm/packages/configcat" target="_blank">ConfigCat's Elixir SDK on Hex.pm</a>
```

Comment:
404, I believe the package is not there yet, just adding a comment so we remember to test before merge.